usaspending_api/recipient/v2/views/list_recipients.py

```diff
@@ -48,6 +48,9 @@ def get_recipients(filters={}, count=None):
         RecipientProfile.objects.filter(qs_filter)
         .values("recipient_level", "recipient_hash", "recipient_unique_id", "recipient_name", amount_column)
         .exclude(recipient_name__in=SPECIAL_CASES)
+        .annotate(
+            uei=Subquery(DUNS.objects.filter(awardee_or_recipient_uniqu=OuterRef("recipient_unique_id")).values("uei"))
```

Comment:
Following up on comments from Slack. This would need to be added to `RecipientProfile` instead of using a Subquery on DUNS since as we phase out the DUNS number it would make the Subquery outdated.